CMake: tbb4py installation failure: Error at x86_64-redhat-linux-gnu/python/cmake_install.cmake:46

@kloczek reported installation issue in scope of https://github.com/oneapi-src/oneTBB/issues/370#issuecomment-939169974 

> cmake 3.21.3 and here is set on cmake parames which I'm using (from script generated by rpmbuild):
```console
+ /usr/bin/cmake -B x86_64-redhat-linux-gnu -D BUILD_SHARED_LIBS=ON -D CMAKE_AR=/usr/bin/gcc-ar -D CMAKE_BUILD_TYPE=RelWithDebInfo -D CMAKE_C_FLAGS_RELEASE=-DNDEBUG -D CMAKE_CXX_FLAGS_RELEASE=-DNDEBUG -D CMAKE_Fortran_FLAGS_RELEASE=-DNDEBUG -D CMAKE_INSTALL_PREFIX=/usr -D CMAKE_NM=/usr/bin/gcc-nm -D CMAKE_RANLIB=/usr/bin/gcc-ranlib -D CMAKE_VERBOSE_MAKEFILE=ON -D DBUILD_SHARED_LIBS=ON -D INCLUDE_INSTALL_DIR=/usr/include -D LIB_INSTALL_DIR=/usr/lib64 -D LIB_SUFFIX=64 -D SHARE_INSTALL_PREFIX=/usr/share -D SYSCONF_INSTALL_DIR=/etc -S . -D TBB4PY_BUILD=ON -D TBB_EXAMPLES=ON -D TBB_TEST=ON -D TBB_TEST_SPEC=ON
```

Installation log:
```
+ cd oneTBB-2021.4.0
+ DESTDIR=/home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64
+ /usr/bin/cmake --install x86_64-redhat-linux-gnu
-- Install configuration: "RelWithDebInfo"
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/blocked_range.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/blocked_range2d.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/blocked_range3d.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/blocked_rangeNd.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/cache_aligned_allocator.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/collaborative_call_once.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/combinable.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/concurrent_hash_map.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/concurrent_lru_cache.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/concurrent_map.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/concurrent_priority_queue.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/concurrent_queue.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/concurrent_set.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/concurrent_unordered_map.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/concurrent_unordered_set.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/concurrent_vector.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_aggregator.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_aligned_space.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_allocator_traits.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_assert.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_concurrent_queue_base.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_concurrent_skip_list.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_concurrent_unordered_base.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_config.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_containers_helpers.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_exception.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_export.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_flow_graph_body_impl.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_flow_graph_cache_impl.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_flow_graph_impl.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_flow_graph_indexer_impl.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_flow_graph_item_buffer_impl.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_flow_graph_join_impl.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_flow_graph_node_impl.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_flow_graph_node_set_impl.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_flow_graph_nodes_deduction.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_flow_graph_tagged_buffer_impl.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_flow_graph_trace_impl.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_flow_graph_types_impl.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_hash_compare.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_intrusive_list_node.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_machine.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_mutex_common.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_namespace_injection.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_node_handle.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_pipeline_filters.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_pipeline_filters_deduction.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_range_common.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_rtm_mutex.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_rtm_rw_mutex.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_scoped_lock.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_segment_table.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_small_object_pool.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_string_resource.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_task.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_task_handle.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_template_helpers.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_utils.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/detail/_waitable_atomic.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/enumerable_thread_specific.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/flow_graph.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/flow_graph_abstractions.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/global_control.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/info.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/memory_pool.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/mutex.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/null_mutex.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/null_rw_mutex.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/parallel_for.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/parallel_for_each.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/parallel_invoke.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/parallel_pipeline.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/parallel_reduce.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/parallel_scan.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/parallel_sort.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/partitioner.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/profiling.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/queuing_mutex.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/queuing_rw_mutex.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/rw_mutex.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/scalable_allocator.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/spin_mutex.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/spin_rw_mutex.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/task.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/task_arena.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/task_group.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/task_scheduler_observer.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/tbb_allocator.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/tbbmalloc_proxy.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/tick_count.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/oneapi/tbb/version.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/blocked_range.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/blocked_range2d.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/blocked_range3d.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/blocked_rangeNd.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/cache_aligned_allocator.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/collaborative_call_once.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/combinable.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/concurrent_hash_map.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/concurrent_lru_cache.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/concurrent_map.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/concurrent_priority_queue.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/concurrent_queue.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/concurrent_set.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/concurrent_unordered_map.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/concurrent_unordered_set.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/concurrent_vector.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/enumerable_thread_specific.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/flow_graph.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/flow_graph_abstractions.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/global_control.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/info.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/memory_pool.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/null_mutex.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/null_rw_mutex.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/parallel_for.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/parallel_for_each.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/parallel_invoke.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/parallel_pipeline.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/parallel_reduce.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/parallel_scan.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/parallel_sort.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/partitioner.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/profiling.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/queuing_mutex.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/queuing_rw_mutex.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/scalable_allocator.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/spin_mutex.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/spin_rw_mutex.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/task.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/task_arena.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/task_group.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/task_scheduler_observer.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/tbb.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/tbb_allocator.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/tbbmalloc_proxy.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/tick_count.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/include/tbb/version.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/lib64/cmake/TBB/TBBTargets.cmake
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/lib64/cmake/TBB/TBBTargets-relwithdebinfo.cmake
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/lib64/cmake/TBB/TBBConfig.cmake
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/lib64/cmake/TBB/TBBConfigVersion.cmake
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/lib64/libtbb.so.12.4
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/lib64/libtbb.so.12
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/lib64/libtbb.so
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/lib64/pkgconfig/tbb.pc
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/lib64/libtbbmalloc.so.2.4
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/lib64/libtbbmalloc.so.2
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/lib64/libtbbmalloc.so
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/lib64/libtbbmalloc_proxy.so.2.4
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/lib64/libtbbmalloc_proxy.so.2
-- Set runtime path of "/home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/lib64/libtbbmalloc_proxy.so.2.4" to ""
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/lib64/libtbbmalloc_proxy.so
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/lib64/libtbbbind_2_5.so.3.4
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/lib64/libtbbbind_2_5.so.3
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/tbb-2021.4.0-2.fc35.x86_64/usr/lib64/libtbbbind_2_5.so
CMake Error at x86_64-redhat-linux-gnu/python/cmake_install.cmake:46 (file):
  file INSTALL cannot find
  "/home/tkloczko/rpmbuild/BUILD/oneTBB-2021.4.0/x86_64-redhat-linux-gnu/python/./build":
  No such file or directory.
Call Stack (most recent call first):
  x86_64-redhat-linux-gnu/cmake_install.cmake:83 (include)
```

--- a/python/CMakeLists.txt
+++ b/python/CMakeLists.txt
@@ -34,6 +34,7 @@ string(STRIP "${TBB4PY_INCLUDE_STRING}" TBB4PY_INCLUDE_STRING)
 
 add_custom_target(
     python_build
+    ALL
     DEPENDS tbb python_copy
     COMMAND
     ${PYTHON_EXECUTABLE} ${PYTHON_BUILD_WORK_DIR}/setup.py

